packages/api/emails/decline/html.pug

```diff
@@ -0,0 +1,28 @@
+div.main
+  img(src=`${process.env.APP_URL}/email/header.jpg` alt='header' style='width:100%; height:auto;')
```

Comment:
The `process.env.APP_URL` might be better suited to be passed into the template similar to how you do with `name` or `patientName` as to keep with consistency for passing dynamic values into this template.